### DISCUSSION TITLE
Passing args over command line ?

### DISCUSSION THREAD

**Discussion by Russsgithub** [savonet/liquidsoap#2366] [Q&A] [answered]
Is it possible to pass args at liquidsoap cli.

e.g.
```liquidsoap test.liq false```

to pass a bool into the script at startup?

I can't find anything in the docs that suggest this is doable.

Thanks

**Comment by smimram** [comment, marked as answer]
Arguments are retrieved with `argv` for instance we can have the following `test.liq`:
```
print("the argument is #{argv(1)}")
```
When you run it with liquidsoap you need to put `--` on the commandline to specify when arguments are starting (before that arguments are taken as script to execute). You can therefore run
```
liquidsoap test.liq -- bla
```
and this will display
```
the argument is bla
```
If you want a boolean you can simply do something like `argv(1) == "true"`.

**Reply by Russsgithub** [reply]
Thanks @smimram
